packages/charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -128,6 +128,9 @@ export class LineChartBase extends React.Component<ILineChartProps, ILineChartSt
       target: this.state.refSelected,
       isBeakVisible: false,
       gapSpace: 15,
+      onDismiss: this._closeCallout,
+      preventDismissOnLostFocus: true,
```

Comment:
why preventdismissonlostfocus?

Reply:
While accessing through keyboard,
When focus shifts from one circle to another circle, we should show callout. If 'preventDismissOnLostFocus' false, then callout will be dismissed. To prevent this sending as true.

Reply:
updated deprecated property